src/Symfony/Component/Security/Http/RateLimiter/DefaultLoginRateLimiter.php

```diff
@@ -41,7 +41,7 @@ protected function getLimiters(Request $request): array
     {
         return [
             $this->globalFactory->create($request->getClientIp()),
-            $this->localFactory->create($request->attributes->get(Security::LAST_USERNAME).'-'.$request->getClientIp()),
+            $this->localFactory->create(strtolower($request->attributes->get(Security::LAST_USERNAME)).'-'.$request->getClientIp()),
```

Comment:
Yeah I am also unsure. IMO if you do allow non-ascii usernames (which doesn't sound that far fetched IMO if not working with emails as identifier) it would be nice if it still protects you as best as it can. mbstring isn't that crazy a requirement, but it could be an optional req too if it's considered too much.

Reply:
I'm happy to send a PR changing that if you agree it'd be better, but I don't want to waste anyone's time otherwise.

Reply:
security-http already comes with symfony/polyfill-mbstring as a transitive dep (because it requires property-access which requires symfony/string) so having it as a root requirement seems fine